src/rpc/methods/eth.rs

```diff
@@ -1958,6 +1958,27 @@ impl RpcMethod<1> for EthNewFilter {
     }
 }
 
+pub enum EthNewPendingTransactionFilter {}
+impl RpcMethod<0> for EthNewPendingTransactionFilter {
+    const NAME: &'static str = "Filecoin.EthNewPendingTransactionFilter";
+    const NAME_ALIAS: Option<&'static str> = Some("eth_newPendingTransactionFilter");
+    const PARAM_NAMES: [&'static str; 0] = [];
+    const API_PATHS: ApiPaths = ApiPaths::V1;
+    const PERMISSION: Permission = Permission::Read;
```

Comment:
Given that this method changes something internal to the node, shouldn't it be `write`? What about other methods we've introduced?

Reply:
as per https://github.com/filecoin-project/lotus/blob/master/documentation/en/api-v1-unstable-methods.md#ethnewpendingtransactionfilter
this and the previous methods we introduced have `Read` permission

Reply:
Could you please check with the Lotus team is that the intended design?